src/urllib3/request.py

```diff
@@ -68,6 +69,7 @@ def request(
         body: Optional[_TYPE_BODY] = None,
         fields: Optional[_TYPE_FIELDS] = None,
         headers: Optional[Mapping[str, str]] = None,
+        json: Optional[Any] = None,
```

Comment:
despite `Any` being the go-to solution with mypy unable to recursively type, I found this quite interesting:
https://github.com/python/typing/issues/182#issuecomment-870697089

maybe something to keep in mind

Reply:
Check this https://github.com/python/typing/issues/182#issuecomment-893657366

Reply:
I think we'll go with `Any` for now until typing has made a decision about a standardized JSON type.